.github/CODEOWNERS

```diff
@@ -19,3 +19,4 @@
 /packages/tools/eslint-plugin                   @alber70g
 /packages/tools/heft-rig                        @alber70g
 /packages/tools/remark-plugins                  @webpro
+/packages/tools/e2e-tests                       @ghislain89e
```

Comment:
Typo? 
```suggestion
/packages/tools/e2e-tests                       @ghislain89
```

Reply:
Yes, it would appear I am sometimes an idiot :D